test/components/ledgerPanelTest.js

```diff
@@ -95,3 +95,62 @@ describe('Payments Panel', function () {
     })
   })
 })
+
+describe('synopsis', function () {
+  function * newFrame (client, frameKey = 2) {
+    yield client
+      .ipcSend('shortcut-new-frame')
+      // wait for correct urlInput based on frameKey
+      .waitUntil(function () {
+        return this.getTabCount().then((count) => {
+          return count === frameKey
+        })
+      })
+      .windowByUrl(Brave.browserWindowUrl)
+      .waitForVisible('div[id="navigator"][data-frame-key="' + frameKey + '"] ' + urlInput)
+      .waitForElementFocus(urlInput)
+  }
+
+  Brave.beforeAll(this)
+
+  before(function * () {
+    yield setup(this.app.client)
+    yield this.app.client
+      .tabByIndex(0)
+      .loadUrl(prefsUrl)
+      .waitForVisible(paymentsTab)
+      .click(paymentsTab)
+      .waitForVisible(paymentsWelcomePage)
+      .click(walletSwitch)
+      .waitUntil(function () {
+        return this.getText(paymentsStatus).then((val) => val.includes('Creating'))
+      })
+      .waitUntil(function () {
+        return this.getText(addFundsButton).then((val) => val.includes('Add funds'))
+      })
+  })
+
+  it('no table if empty synopsis', function * () {
+    yield this.app.client
+      .isExisting(ledgerTable).then((isExisting) => isExisting === false)
+  })
+
+  it.skip('creates synopsis table after visiting a site', function * () {
+    // TODO (mrose17): re-enable this test once ledger properly ignores the 8
+    // second pageview limit during tests
+    yield this.app.client
+      .url('http://web.mit.edu/zyan/Public/wait.html')
+      .waitUntil(function () {
+        return this.getText('div').then((val) => val === 'done')
+      })
+      .windowByUrl(Brave.browserWindowUrl)
```

Comment:
@mrose17 this test seems to not pass because the ledger is still requiring us to be on the page for 8 seconds even though `process.env.NODE_ENV === 'test'`. i.e., if you add a `.debug()` here and wait for at least 8 seconds, then the page shows up on about:preferences as expected.